### PR TITLE
Allow to request the originals as well

### PR DESCRIPTION
**Pull request by rogierslag**
@alexnederlof @wouter-willems Ready for review!

### CODE REVIEW

**Comment by alexnederlof** [comment]
:+1: for code, but it would really help if you would add doc with example URLs so I can actually follow what all those matches etc do

File: resize.js [hunk 1]
```diff
@@ -105,16 +105,25 @@ function splitUrl(url) {
...
+           resolutionY: parseInt(matches[3], 10),
+           fileType: matches[6].toLowerCase()
+        };
+    
+        if (matches[5] !== undefined) {
+            res.resolutionX *= parseInt(matches[5], 10);
+            res.resolutionY *= parseInt(matches[5], 10);
+        }
+    } else {
+        matches = url.match(/^\/(.*)\.([^.]+)$/);
```
Comment (wouter-willems) [inline resize.js]: always put examples along with regexes

File: resize.js [hunk 2]
```diff
@@ -261,6 +270,27 @@ image.upload = function (req, res) {
         res.end();
     }
 };
+image.getOriginal = function (req, res) {
+    var matches = req.url.match(/^\/(.*)\.([^.]+)$/);
```
Comment (wouter-willems) [inline resize.js]: isnt this the same as the above? `var` it

File: resize.js [hunk 3]
```diff
@@ -324,7 +354,9 @@ function startServer() {
     app.use(bodyParser.json());
     app.use(responseTime(logRequest));
     app.get('/healthcheck', serverStatus);
-    app.get('/*', image.get);
+    app.get('/*_*_*_*x.*', image.get);
```
Comment (wouter-willems) [inline resize.js]: what the hell does this match, lol

**Pull request merged**